ivy/functional/frontends/numpy/manipulation_routines/changing_kind_of_array.py

```diff
@@ -0,0 +1,12 @@
+# local
+import ivy
+
+
+def asarray(obj,
```

Comment:
On the docs for numpy this is 'a' instead of 'obj' we should keep it as close to their implementation as possible :)

Reply:
Okay, I got "obj" from array api standard but I'll keep it as numpy docs as you requested.